lib/ui/cachedblock.js

```diff
@@ -0,0 +1,55 @@
+const Block = require('./block');
+const { blockSize } = require('./variables');
+
+class CachedBlock extends Block {
```

Comment:
Sure, but with React or re:dom coming up we might just want to deprecate this class soon.